canonical-web-and-design/maas-ui
# Create Storage component in Settings

## Done
- Created MAAS config selectors, reducers and actions
- Created non-persistent Storage component in settings. Saving the form will be implemented in canonical-web-and-design/MAAS-squad#1389

## QA
- Check that config is loaded into state on load (in Redux Dev Tools for example)
- Check that the React storage settings match those in the Django version.
- Check that changing the Django settings correctly updates the React settings on reload.
- Check that a warning shows up for blank and vmfs6 storage layouts.
- Check that clicking save displays an alert with the currently set values.
